forms-flow-web/src/components/Form/EditForm/FormSettings.js

```diff
@@ -178,7 +209,9 @@ const FormSettings = forwardRef((props, ref) => {
           dataTestid="form-name"
           name="title"
           ariaLabel={t("Form Name")}
-        />
+          onBlur={() => validateField('name', formDetails.title)}
+          />
+          {errors.name && <div className="validation-text">{errors.name}</div>}
```

Comment:
This FormInput already have a  feedback message or "error text". checkout  Forminput component and update.

Reply:
Updated